sdk/test-utils/recorder/test/testProxyClient.spec.ts

```diff
@@ -169,8 +170,8 @@ describe("TestProxyClient functions", () => {
           await client.start({ envSetupForPlayback: {} });
           throw new Error("should not have reached here, start() call should have failed");
         } catch (error: any) {
-          expect((error as RecorderError).name).to.equal("RecorderError");
-          expect((error as RecorderError).message).to.equal(
+          assert.equal((error as RecorderError).name, "RecorderError");
```

Comment:
total nitpicking, and feel free to ignore since it's also existing code, but I don't believe this `error as RecorderError` gives us anything here (it's an `any` today, and `as` conversions aren't checked). Would be nice to do a minor cosmetic cleanup while we're here